# Phylochemical mapping of non-proteogenic amino acids

![workflow schematic](image.png "cover_image")

## Overview
This project aims to elucidate the phylogenetic distribution of plant natural products by focusing on an important class of metabolites called non-proteogenic amino acids. We've created a workflow to map non-proteogenic amino acids onto the plant tree of life, revealing lineage-specific compound distributions and providing insights into plant chemical diversity. This repository hosts the raw data and R code necessary to reproduce our findings.

## Data
Contained within this repository is the complete set of raw data used in our analyses. The data are structured to facilitate easy replication of our results and further phylochemical distributions of (plant) metabolites.

## R Code
The R scripts provided enable the recreation of our results from the ground-up. They are extensively commented to aid in understanding and can be adapted for extended analyses or related projects.

## Results
By mining and manually curating over 550 compound-species associations from peer-reviewed scientific literature, we have created a phylochemical map that highlights the distribution of non-proteogenic amino acids at various taxonomic levels. This is proof-of-concept approach can be easily applied to other classes of (plant) natural products.
